src/interfaces/messages/types.ts

```diff
@@ -0,0 +1,24 @@
+import type { BaseMessageReply } from '../../core/message-reply.js';
+import type { BaseMessage, Descriptor } from '../../core/types.js';
+import type { DwnInterfaceName, DwnMethodName } from '../../core/message.js';
+
+export type MessagesGetDescriptor = Pick<Descriptor, 'interface' | 'method' > & {
```

Comment:
Just thinking out loud: I don't like `Pick`s very much. Would prefer to reduce the fields in `Descriptor` and move non-universal fields into message-specific descriptor types. But that is outside the scope of this PR.

Reply:
I agree! Gave it a shot because it was mentioned in the last PR.